python/src/functions.rs

```diff
@@ -20,7 +20,7 @@ use crate::udf;
 use crate::{expression, types::PyDataType};
 use datafusion::arrow::datatypes::DataType;
 use datafusion::logical_plan;
-use pyo3::{prelude::*, wrap_pyfunction};
+use pyo3::{prelude::*, types::PyTuple, wrap_pyfunction};
```

Comment:
Are the changes to python intended for this PR @Jimexist ?

Reply:
not really, let me rebase.